frontend/src/main-components/StudyModeGameHandler.js

```diff
@@ -0,0 +1,42 @@
+let quiz = [];
```

Comment:
Globals are a bad idea -- can we put this state into a class instead?

Reply:
done.